bin/cml/publish.js

```diff
@@ -43,7 +43,7 @@ exports.builder = (yargs) =>
         type: 'boolean',
         alias: 'gitlab-uploads',
         description:
-          "Uses driver's native capabilities to upload assets instead of CML's storage. Currently only available for GitLab CI."
+          "Uses driver's native capabilities to upload assets instead of CML's storage. Not available for GitHub."
```

Comment:
much minor such wow

```suggestion
          "Uses driver's native capabilities to upload assets instead of CML's storage. Not available on GitHub."
```

Reply:
🐶 I was about to type “on” but thought “if it says for, it must be correct” 🙃